README.md

```diff
@@ -58,7 +58,7 @@ The Flutter embedder for Tizen.
 ### Notes
 
 - To build an app (TPK) with the embedder generated in the above, copy the output artifacts (`libflutter_tizen*.so`) into the [flutter-tizen](https://github.com/flutter-tizen/flutter-tizen) tool's cached artifacts directory (`flutter/bin/cache/artifacts/engine`) and run `flutter-tizen run` or `flutter-tizen build tpk`.
-- To use the target device's `libstdc++.so` instead of the embedder's built-in libc++ (`third_party/libcxx`), provide the `--system-cxx` option to `tools/gn`. Tizen 5.5's `libstdc++.so` is not compatible with C++17 so you must use a sysroot for Tizen 6.0 or above.
+- To use the target device's `libstdc++.so` instead of the embedder's built-in libc++ (`third_party/libcxx`), provide the `--system-cxx` option to `tools/gn`. If Tizen version is 6.0 or above, you must use a sysroot.
```

Comment:
It's actually
```suggestion
- To use the target device's `libstdc++.so` instead of the embedder's built-in libc++ (`third_party/libcxx`), provide the `--system-cxx` option to `tools/gn`. Tizen 5.5's `libstdc++.so` is not compatible with C++17, so you cannot use this option for Tizen 5.5.
```
or you can simply remove the sentence (not very important information).

Reply:
I have excluded code that changes the default version from this patch.
I will remove this sentence when excluding version 5.5 in the future.